src/client/components/pages/about.js

```diff
@@ -110,7 +115,7 @@ function AboutPage() {
 			<p>
 				The core components of BookBrainz are the site, the
 				webservice and the database. All of these components are
-				running on MetaBrainz's Sakura server, which also runs&nbsp;
+				running on MetaBrainz&#39;s Sakura server, which also runs&nbsp;
```

Comment:
Leaving this as it was is fine, but it seems that GitHub changed what I meant to say - that using `&rsquo;` is better (rather than `'` or `&#39;`).